### PR TITLE
maven: add openjdk 15 & 16 slim

### PR DESCRIPTION
**Pull request by carlossg**
Add git to openjdk images

### CODE REVIEW

**Comment by tianon** [comment]
Looks like CI is pointing out that AdoptOpenJDK is now Focal-only (no more Bionic tags) and that OpenJDK 14 is EOL (although not AdoptOpenJDK 14 "yet").

**Comment by carlossg** [comment]
not sure why naughty is failing, when I run locally `./naughty-from.sh maven` there is no output

File: library/maven [hunk 1]
```diff
@@ -83,27 +78,37 @@ Directory: adoptopenjdk-11
...
+GitCommit: ecb160edac810c845116ef70ffbda8bfeca21075
+Directory: adoptopenjdk-15
+
+Tags: 3.6.3-adoptopenjdk-15-openj9, 3.6-adoptopenjdk-15-openj9, 3-adoptopenjdk-15-openj9
+Architectures: amd64, ppc64le, s390x
+GitCommit: b243010db790a1ab8f6a43d26ae86a76cba86b55
+Directory: adoptopenjdk-15-openj9
+
 Tags: 3.6.3-adoptopenjdk-8, 3.6-adoptopenjdk-8, 3-adoptopenjdk-8
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
```
Comment (yosifkit) [inline library/maven:105]: The failure is related to this version,`maven:3.6.3-adoptopenjdk-8` being `FROM adoptopenjdk:8-jdk-hotspot`, which no longer has an arm32v7 image (https://github.com/docker-library/official-images/pull/8953).

Comment (carlossg) [reply library/maven]: I just needed to pull master 🤦

**Pull request merged**